packages/client/hmi-client/src/page/Home.vue

```diff
@@ -170,15 +170,15 @@
 
 <script setup lang="ts">
 import { computed, onMounted, ref } from 'vue';
-import SelectedDocumentPane from '@/components/documents/selected-document-pane.vue';
+import TeraSelectedDocumentPane from '@/components/documents/tera-selected-document-pane.vue';
 import { IProject } from '@/types/Project';
 import { XDDSearchParams } from '@/types/XDD';
 import { DocumentType } from '@/types/Document';
 import { searchXDDDocuments } from '@/services/data';
 import useResourcesStore from '@/stores/resources';
 import useQueryStore from '@/stores/query';
 import ProjectCard from '@/components/projects/ProjectCard.vue';
-import DocumentCard from '@/components/documents/DocumentCard.vue';
+import TeraDocumentCard from '@/components/home/tera-document-card.vue';
```

Comment:
Why should this component be moved, we might use those cards elsewhere

Reply:
We don't use this component elsewhere though, it is quite specific to the home page.